.eslintrc.json

```diff
@@ -29,14 +29,13 @@
     "react/react-in-jsx-scope": ["off"],
     "react/sort-comp": ["off"],
     "semi": ["error", "never"],
-    "space-before-function-paren": ["off"],
+    "space-before-function-paren": ["error", "always"],
```

Comment:
I'm pretty sure we can just remove this rule, since it's inherited from the AirBnB lint config.

Reply:
AirBnB's rules are `["error", "never"]`, which remove the space between the name and the parens.